Symfony/CS/Tests/Fixer/Contrib/OrderedUseFixerTest.php

```diff
@@ -638,4 +653,694 @@ public function provide70Cases()
             ),
         );
     }
+
+    /*
+    |--------------------------------------------------------------------------
+    | Test sorting by length
+    |--------------------------------------------------------------------------
+    */
+
+    protected function setSortByLengthConfiguration()
```

Comment:
I find proxy method pointless. it's calling one method to call another... one not call it directly ?